template.rb

```diff
@@ -234,6 +234,11 @@ def render_file(filename)
   # Set up dotenv
   file ".env.development", render_file(".env.development")
 
+  create_file ".gitignore" do
```

Comment:
What is the purpose of this change? Does it relate to the ruby version?

Reply:
Yes. Since in latest rails `.gitignore` is not included.

Reply:
Really? I still get one:

```
testgitignore HEAD % ls -a
.             .gitignore    Gemfile.lock  app           config.ru     log           storage       vendor
..            .ruby-version README.md     bin           db            package.json  test
.git          Gemfile       Rakefile      config        lib           public        tmp
testgitignore HEAD % rails -v
Rails 5.2.0
```